backend/src/context/rpc.rs

```diff
@@ -279,8 +278,25 @@ impl RpcContext {
             })
             .await?;
         let peek = self.db.peek().await?;
+        let mut all_dependency_config_errs = BTreeMap::new();
```

Comment:
So, the idea is that I want to accumulate all the package dependency errors. For the mutate below, a kind of global reset.